docs/monitoring/metrics.md

```diff
@@ -1031,7 +1031,7 @@ Thus, in order to infer the metric identifier:
       <th rowspan="8">Task</th>
       <td rowspan="4">buffers</td>
       <td>inputQueueLength</td>
-      <td>The number of queued input buffers.</td>
+      <td>The number of queued input buffers. (ignores LocalInputChannels which are using blocking subpartitions)</td>
```

Comment:
`using the queue of subpartitions`? `blocking` is not very accurate here because local channel might correspond to both pipelined and blocking subpartitions.

Reply:
`ignores` -> `Consider`?